js-weather/weather.js

```diff
@@ -0,0 +1,48 @@
+//関数リテラルは呼び出し元のコードよりも先に記述する
+const getWeather = (data) => {
+    if (data === null) {
+        //<div id = "result">HTMLElementオブジェクトを生成
+        const result = document.getElementById('result');
+        result.textContent = '天気を取得できませんでした';
+    } else {
+        // 天気と気温を取得し、配列にする
+        const weatherConditions = data.weather[0].description;
+        const temp = data.main.temp;
+        const MaxTemp = data.main.temp_max;
```

Comment:
```suggestion
        const maxTemp = data.main.temp_max;
```

基本的には、変数名は小文字スタートのキャメルケースですね。
jsでこの記述すると、定数やらクラス名だと思われます。